packages/nvidia_nat_core/src/nat/eval/remote_workflow.py

```
@@ -43,91 +43,112 @@ def __init__(self, config: EvaluationRunConfig, max_concurrency: int):
+                        line = line.decode('utf-8').strip()
+                        if not line:
+                            continue
 
-    async def run_workflow_remote_with_limits(self, session: aiohttp.ClientSession, item: EvalInputItem, pbar: tqdm):
+                        if line.startswith(DATA_PREFIX):
+                            try:
+                                chunk_data: dict = json.loads(line[len(DATA_PREFIX):])
+                                if chunk_data.get("value"):
+                                    final_response = chunk_data.get("value")
+                            except json.JSONDecodeError as e:
+                                logger.exception("Failed to parse generate response chunk: %s", e)
```

Ruff (0.14.14)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)